web-client/src/components/data/dataset_input/DatasetInput.vue

```diff
@@ -150,17 +160,19 @@ import { data } from '@epfml/discojs-core'
 import Upload from '@/assets/svg/Upload.vue'
 import IconCard from '@/components/containers/IconCard.vue'
 import FileSelection from './FileSelection.vue'
+import { useToaster } from '@/composables/toaster'
 
 interface Props {
   task: Task
   datasetBuilder: data.DatasetBuilder<File>
-  isOnlyPrediction: Boolean
+  isOnlyPrediction: Boolean //TODO: seems to always be false
 }
+const toaster = useToaster()
 
 const props = defineProps<Props>()
 
 const csvRows = ref<{ filename: string, label: string }[]>()
-const isBoxView = ref<boolean>(false)
+const connectImagesByGroup = ref<boolean>(true)
```

Comment:
nitpicking 3000: no need to precise the type when it's derived from the argument

```suggestion
const connectImagesByGroup = ref(true)
```